test/e2e/apimachinery/namespace.go

```diff
@@ -246,4 +248,29 @@ var _ = SIGDescribe("Namespaces [Serial]", func() {
 	ginkgo.It("should always delete fast (ALL of 100 namespaces in 150 seconds) [Feature:ComprehensiveNamespaceDraining]",
 		func() { extinguish(f, 100, 0, 150) })
 
+	ginkgo.It("should patch a Namespace", func() {
+		ginkgo.By("creating a Namespace")
+		namespaceName := "nspatchtest"
+		_, err := f.ClientSet.CoreV1().Namespaces().Create(&v1.Namespace{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: namespaceName,
+			},
+		})
+		framework.ExpectNoError(err, "failed creating Namespace")
+
+		ginkgo.By("patching the Namespace")
+		nspatch := fmt.Sprintf(`{"metadata":{"labels":{"testLabel":"testValue"}}}`, namespaceName)
+		_, err = f.ClientSet.CoreV1().Namespaces().Patch(namespaceName, types.StrategicMergePatchType, []byte(nspatch))
+		framework.ExpectNoError(err, "failed to patch Namespace")
+
+		ginkgo.By("get the namespace and ensuring it has the label")
```

Comment:
final minor nit: missed that namespace is lowercase here, which is inconsistent with the rest of the `.By` calls.

Reply:
on the other hand existing code has it lowercase everywhere:
https://github.com/kubernetes/kubernetes/pull/86854/files#diff-35baea3ee2d40a29c405b9114bde6c84R189
so probably better to lowercase it in this PR too.

Reply:
Updated in https://github.com/kubernetes/kubernetes/pull/86854/commits/554eb47e512aedfbf511d621a2584ce5f3559776